libs/components/autonumeric/src/lib/modules/autonumeric/autonumeric.directive.spec.ts

```diff
@@ -207,98 +189,41 @@ describe('Autonumeric directive', () => {
   }));
 
   it('should update numeric value on keyup', fakeAsync(() => {
-    detectChanges();
-
-    const autonumericInstance =
-      // @ts-ignore
-      fixture.componentInstance.autonumericDirective['autonumericInstance'];
-    const spy = spyOn(autonumericInstance, 'getNumber').and.callThrough();
+    detectChangesTick();
 
     const input = fixture.nativeElement.querySelector('input');
 
     input.value = '1000';
     SkyAppTestUtility.fireDomEvent(input, 'input');
     SkyAppTestUtility.fireDomEvent(input, 'keyup');
-    detectChanges();
-
-    expect(spy).toHaveBeenCalled();
-  }));
-
-  it('should not update numeric value on keyup when no change is made', fakeAsync(() => {
```

Comment:
It seems like we could leave this test if we want. You could just test that the input value doesn't change from `1000` to `1,000`.

Reply:
I could go either way here. The two tests were trying to test the same functionality; however, one was testing with the currency option set and one was testing without. So they were technically checking the same thing in different configurations.

Reply:
reverted in: [add back non-redundant test](https://github.com/blackbaud/skyux/pull/606/commits/dea3c560d06a763a25ca4dfe35c24b04f1b9764e)